docs/components/receive.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Receiver
+type: docs
+menu: components
+---
+
+# Receiver
+
+The `thanos receive` command implements the [Prometheus Remote Write API](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write). It builds on top of existing Prometheus servers and retains their usefulness while extending their functionality with long-term-storage, horizontal scalability, and downsampling. The [Thanos Sidecar](./sidecar.md) is not sufficient for this, as the system would always lag the block length behind (typically 2 hours), which would prevent the most common query pattern of Prometheus: real-time queries of very recent data.
+
+This component is only recommended for uses for whom pushing is the only viable solution, for example, analytics use cases or cases where the data ingestion must be client-initiated, such as software as a service type environments.
```

Comment:
```suggestion
We recommend this component to users who can only push into a Thanos due to air-gapped environments. Examples are: analytic use cases or cases where the data ingestion must be client-initiated, such as software as a service type environments.
```

I don't see why the receiver is particularily good for analytic use case compared to the sidecar.

Reply:
Agree

Reply:
At least for now, there is no difference for analytics